### PR TITLE
202 remove nswag code generator

### LINKED ISSUE #202: ✨ OpenAPI/Swagger - Consider switching from NSwag to Swashbuckle
- We are not generating client side code anymore (which was one of the reasons we were using NSwag).
- Now that we are only generating the OpenAPI Spec, is it better to use NSwag or Swashbuckle to generate the spec
- We still want an API UI
- Check with @matt-goldman , @drwharris , Alex Rothwell on feedback/opinions around this
- Enum generation needs to also be considered as this is often a painpoint
- ~Consider using magic strings for endpoint routes~

~Notes~
~- Refit is another .NET library to assist with calling  REST APIs~

### PR DESCRIPTION
**Pull request by wicksipedia**
﻿> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

closes #202 

> 2. What was changed?

Removed NSwag.MSBuild as it was adding to build time but not value

> 3. Did you do pair or ~mob~ ensemble programming?

no

### CODE REVIEW

**Review by william-liebenberg — APPROVED** [review]
💪

File: src/WebApi/WebApi.csproj [hunk 1]
```diff
@@ -6,43 +6,28 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    <!-- as per https://github.com/dotnet/aspnetcore/issues/50836#issuecomment-1819612514  -->
-    <!-- recommendation from msft is to disable AD0001 until first servicing release of .net8   -->
-    <NoWarn>$(NoWarn);AD0001</NoWarn>
+    <UserSecretsId>6893d2b2-792c-4bb6-a737-64aa9cd6e7a6</UserSecretsId>
```
Comment (danielmackay) [inline src/WebApi/WebApi.csproj:9]: User secrets ID needs to be removed

**Review by danielmackay — COMMENTED** [review]
@wicksipedia - One comment.  Can you please open a new PR for this?

File: src/WebApi/WebApi.csproj (hunk 1 above)
Comment (wicksipedia) [reply src/WebApi/WebApi.csproj:9]: https://github.com/SSWConsulting/SSW.CleanArchitecture/pull/235

**Pull request merged**